### PR TITLE
Added landing page

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Screenshot
Screenshot (190)

### CODE REVIEW

File: src/pages/Landing/Landing.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.welcome-content {
+  color: var(--secondary-text);
+}
+
+@media screen and (max-width: 37.5em) {
+  main.grid-2-col {
+    grid-template-columns: 1fr;
+  }
```
Comment (hrishikeshmane) [inline src/pages/Landing/Landing.css:34]: suggestion: you can use BEM convention for css classes.

**Review by hrishikeshmane — COMMENTED** [review]
Looks good.

File: src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          </a>
+          <a
+            className="m-1"
+            href="https://www.instagram.com/sanket_dhabarde/"
+            target="_blank"
+          >
+            <i className="fab fa-instagram"></i>
+          </a>
+          <a
+            className="m-1"
```
Comment (Aditi0505) [inline src/components/Footer/Footer.jsx:38]: You can create a common component for it and pass changing values as props, or you can create a list of object and iterate it, to minimise no. of lines of code.

**Review by Aditi0505 — APPROVED** [review]
Just left one suggestion. Rest looks good.
- Good folder structure.
- Right practice of having static images in public folder. 
- Good naming conventions followed.

File: src/components/Footer/Footer.jsx (hunk 2 above)
Comment (SanketDhabarde) [reply src/components/Footer/Footer.jsx:38]: Actually. I am not going to reuse this. So thought of keeping it this way without creating components. But it makes sense to create components. I will do it thanks for the suggestion.

File: src/pages/Landing/Landing.css (hunk 1 above)
Comment (SanketDhabarde) [reply src/pages/Landing/Landing.css:34]: I can. but I have not used BEM in my component library. So to make it consistent I am writing classes with the kabab case. Thanks for the suggestion.

**Pull request merged**